rest_api/openapi.yaml

```diff
@@ -592,6 +662,52 @@ definitions:
         type: string
         format: binary
 
+  TransactionReceipt:
```

Comment:
This should be `TransactionReceipts` and wrapped in an array, since the route will return many receipts. 

Just replace this line with:
```
  TransactionReceipts:
    type: array
    items:
```
And then increase the indentation of everything else.

(also update the two `$ref` above)